### PR TITLE
Feature/multilanguage public interface

### PR DESCRIPTION
**Pull request by drewpotter**
This is the initial work for adding translations using yaml for the possible answers for both the demographic and normal (multiple choice) questions.

### CODE REVIEW

File: app/controllers/responses_controller.rb [hunk 1]
```diff
@@ -19,8 +19,10 @@ def index
   end
 
   def new
+    # set the locale based upon the default_langauge of the survey
     I18n.locale = @survey.settings["default_language"].to_sym || I18n.default_locale
```
Comment (WKocur) [inline app/controllers/responses_controller.rb]: Is this only for the public side of the interface? This means that if the user completes the survey in a certain language, by selecting this in the dropdown, the results page won't necessarily be shown in that language. Is this correct and intended?

**Comment by WKocur** [comment]
Screenshot 2019-09-25 at 10 27 02
I get this error when trying to take the survey.

**Comment by drewpotter** [comment]
Many thanks @WKocur it looks like it isn't seeding properly for some reason. Please can you run `rails db:seed` and try again?

File: app/controllers/responses_controller.rb (hunk 1 above)
Comment (drewpotter) [reply app/controllers/responses_controller.rb]: This is a good point... let me double check! Thanks for pointing this out 😄

**Pull request merged**